alibi/explainers/ale.py

```diff
@@ -103,6 +111,13 @@ def explain(self, X: np.ndarray, features: Optional[List[int]] = None, min_bin_p
         min_bin_points
             Minimum number of points each discretized interval should contain to ensure more precise
             ALE estimation.
+        grid_points
+            Custom grid points. Must be a `dict` where the keys features indices  and the values are
+            `numpy` arrays defining the grid points for each feature.
+
+        tol
+            Tolerance to delete consecutive quantiles obtained after clipping their values between the minimum and
+            maximum feature values. Only used when `feature_grid_points` is not ``None``.
```

Comment:
`feature_grid_points` -> `grid_points`

Reply:
I'm not 100% sure what this does from the description (quantiles + grid_points + clipping?), would be great to clarify.